### DISCUSSION TITLE
Can't open files inside wsl with `O`

### DISCUSSION THREAD

**Discussion by LinArcX** [gokcehan/lf#1435] [Q&A] [answered]
Hello, i setup wsl 2 (Debian Buster) on windows 10 and i'm using lf as file manager in it. the only problem that I have right now is that I can't open files with lf inside wsl. This is the content of my lfrc:
```
map <enter> open

cmd open ${{
	case $(file --mime-type -Lb $f) in
		text/x-shellscript) $EDITOR $fx;;
        *) for f in $fx; do $OPENER $f > /dev/null 2> /dev/null & done;;
  esac
}}


```

And this is the error that I receive:
```
: not found
--: 5: --: Syntax error: word unexpected (expecting "in")
```

**Comment by gokcehan** [comment, marked as answer]
@LinArcX This example works for me. What is your `shell`? Please make sure you try this with a minimal `lfrc` or post everyhing in your `lfrc`.
